soc/nxp/lpc/lpc55xxx/Kconfig.soc

```diff
@@ -0,0 +1,121 @@
+# Copyright 2019, 2023-2024 NXP
+# SPDX-License-Identifier: Apache-2.0
+
+config SOC_SERIES_LPC55XXX
+	bool
+	select SOC_FAMILY_LPC
+
+config SOC_SERIES
+	default "lpc55xxx" if SOC_SERIES_LPC55XXX
+
+config SOC_LPC55S06
+	bool
+	select SOC_SERIES_LPC55XXX
+
+config SOC_LPC55S16
+	bool
+	select SOC_SERIES_LPC55XXX
+
+config SOC_LPC55S28
+	bool
+	select SOC_SERIES_LPC55XXX
+
+config SOC_LPC55S36
+	bool
+	select SOC_SERIES_LPC55XXX
+
+config SOC_LPC55S69
+	bool
+	select SOC_SERIES_LPC55XXX
+
+config SOC_LPC55S69_CPU0
+	bool
+	select SOC_LPC55S69
+
+config SOC_LPC55S69_CPU1
+	bool
+	select SOC_LPC55S69
+
+config SOC
+	default "lpc55s69_cpu0" if SOC_LPC55S69_CPU0
+	default "lpc55s69_cpu1" if SOC_LPC55S69_CPU1
+	default "lpc55s06" if SOC_LPC55S06
+	default "lpc55s16" if SOC_LPC55S16
+	default "lpc55s28" if SOC_LPC55S28
+	default "lpc55s36" if SOC_LPC55S36
+
+config SOC_PART_NUMBER_LPC55S06JBD64
+	bool
+
+config SOC_PART_NUMBER_LPC55S16JBD64
+	bool
+
```

Comment:
Not that it needs updating here but the other nxp pr does things differently in that a part number selects the soc, and means a board file just selects the part number, would be good to have things in unison across socs for doing this

Reply:
Agreed